### ISSUE TITLE
Parentheses placement for long calls

### ISSUE THREAD

**Issue by dzharkov** [yole/kotlin-style-guide#31]
```kotlin
foo(
   bar(), baz(), 
   foobarbaz(
       arg1, arg2, ...
   )
)
```

**Comment by dzharkov** [comment]
Lisp-like
```kotlin
foo(
   bar(), baz(), 
   foobarbaz(
       arg1, arg2, ..))
```

**Comment by damianw** [comment]
Isn't this already addressed in #8? (https://github.com/yole/kotlin-style-guide/issues/8#issuecomment-223011004)

**Comment by dzharkov** [comment]
@damianw It is. I was sure there should be something similar but didn't manage to find it
